wavefront_python_sdk/entities/histogram/histogram_granularity.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function
+
+from enum import Enum
+
+
+class HistogramGranularity(Enum):
```

Comment:
I'd probably go with module constants instead of having an enum here.
You already have a module named `histogram_granularity`, and using it's constants would be as simple as `histogram_granularity.MINUTE` instead of initializing an enum like `histogram_granularity.HistogramGranularity(histogram_granularity.HistogramGranularity.MINUTE)`